src/js/components/tasks/createForm.tsx

```diff
@@ -203,9 +210,9 @@ const CreateTaskModal = ({
           type="submit"
           label={
             isSaving ? (
-              <LabelWithSpinner label={i18n.t('Adding…')} variant="inverse" />
+              <LabelWithSpinner label={i18n.t('Creating…')} variant="inverse" />
             ) : (
-              i18n.t('Add')
+              i18n.t('Create')
```

Comment:
@oluoluoxenfree When we don't have an epic, we want some additional help text at the top of the modal (maybe a `p` inside the top of the `form`?).